src/Sylius/Component/Grid/Definition/Field.php

```diff
@@ -152,7 +152,11 @@ public function setEnabled($enabled)
      */
     public function setSortable($sortable)
     {
-        $this->sortable = $sortable ?: $this->name;
+        if ($sortable === true) {
+            $this->sortable = $this->name;
+        }
+
+        $this->sortable = $sortable;
```

Comment:
My question is , why change that line ? 

> Expression expr1 ?: expr3 returns expr1 if expr1 evaluates to TRUE, and expr3 otherwise.

That's from the PHP documentation.

Also as @Zales0123 is saying with that code `$this->soratable` will always be `$sortable`.